platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/SwirldsPlatform.java

```diff
@@ -675,6 +646,19 @@ public class SwirldsPlatform implements Platform {
         platformWiring.wireExternalComponents(
                 platformStatusManager, appCommunicationComponent, transactionPool, latestCompleteState);
 
+        // wire ISS output
+        final IssHandler issHandler =
+                new IssHandler(stateConfig, this::haltRequested, this::handleFatalError, issScratchpad);
+        final OutputWire<IssNotification> issOutput =
+                platformWiring.getIssDetectorWiring().issNotificationOutput();
+        issOutput.solderTo("issNotificationEngine", n -> notificationEngine.dispatch(IssListener.class, n));
+        issOutput.solderTo("statusManager", n -> {
+            if (Set.of(IssType.SELF_ISS, IssType.CATASTROPHIC_ISS).contains(n.getIssType())) {
+                platformStatusManager.submitStatusAction(new CatastrophicFailureAction());
```

Comment:
You aren't submitting a `CatastrophicFailureAction` for `IssType.OTHER_ISS`, but you *are* halting the node in that case in the `IssHandler`. Additionally, the `IssHandler` only halts the node if the `haltOnAnyIss` config value is `true`, yet the status update here is always going out, which will kill the node.

These two side effects of an ISS should always be occurring in lockstep.